common/djangoapps/third_party_auth/models.py

```diff
@@ -315,6 +318,13 @@ def clean(self):
             raise ValidationError('Either an icon class or an icon image must be given (but not both)')
 
     @property
+    def enforce_data_sharing_consent(self):
+        """
+        Does the provider require data sharing consent?
```

Comment:
Need to update docstring here -- also it would be good to define 'enforce' and how it differs from 'require'